### ISSUE TITLE
CANS-F Did not Complete status

### ISSUE THREAD

**Issue by jessicamorris64** [cssat/sprout-issues#199]
As a provider I need a button created for "did not complete" ; that allows providers to bypass the CANS-F and complete the End of Intervention Summary. 
Please contact Delton for more info

**Comment by tinhart** [comment]
@subaykan, after our mtg with Kevin and Kirstin from DCYF yesterday, 10/17, it sounds like this CANS-F issue is complete. Can we close this issue, or does it actually represent more/different work than what we closed yesterday?

**Comment by subaykan** [comment]
No, this is not part of the recently completed CANS-F work. It is a separate ask.
